src/overlays/actors/ovl_Shot_Sun/z_shot_sun.c

```diff
@@ -90,14 +90,18 @@ void ShotSun_SpawnFairy(ShotSun* this, PlayState* play2) {
         CutsceneManager_Stop(this->actor.csId);
         switch (params) {
             case SHOTSUN_FAIRY_SPAWNER_SUNS:
-                fairyType = ENELF_TYPE_7;
+                fairyType = FAIRY_TYPE_7;
```

Comment:
... You could get a big fairy by shooting the sun in OOT?

Reply:
Not quite but close. There's the small platform where the fire arrows drop down after shooting the sun, the same platform where there's a gold skull as child. If you go to that spot and play suns song, then yes a big fairy will spawn. I'm assuming this was named because it's directly below where you shoot the sun